website/docs/workloads/sysbench/sysbench-oltp-profiles.md

```diff
@@ -20,14 +20,14 @@ idea. The name of the client must match the name of the system or the value of t
 
```

Comment:
Combine the documentation for ALL profiles for Sysbench into a single markdown document for simplicity of finding that documentation. In this case you can have sections describing both the PERF-MYSQL-OLTP-SYSBENCH and PERF-MYSQL-TPCC-SYSBENCH profile documentation in this 1 document.